test/functional/Java16andUp/playlist.xml

```diff
@@ -116,6 +116,32 @@
 			-excludegroups $(DEFAULT_EXCLUDE); \
 			$(TEST_STATUS)
 		</command>
+		<platformRequirements>bits.64,^arch.ppc,^arch.390,^arch.riscv,^os.zos,^os.sunos,^arch.arm</platformRequirements>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>16+</subset>
+		</subsets>
+	</test>
+	
+	<test>
+		<testCaseName>Jep389Tests_testClinkerFfi</testCaseName>
```

Comment:
Is this test specific to openj9/ibm? If so, please add `<impl>`. Thanks

Reply:
Nope. It is intended for OpenJ9/OpenJDK at least for now.